redisgraph-bechmark-go.go

```diff
@@ -33,7 +33,17 @@ func main() {
 	// in the meantime added this two fake vars
 	var loopV = false
 	var loop *bool = &loopV
+	version := flag.Bool("v", false, "Output version and exit")
 	flag.Parse()
+	git_sha := toolGitSHA1()
+	git_dirty_str := ""
+	if toolGitDirty() {
+		git_dirty_str = "-dirty"
+	}
+	log.Printf("redisgraph-benchmark-go (git_sha1:%s%s)\n", git_sha, git_dirty_str)
+	if *version {
+		os.Exit(0)
```

Comment:
I think you want a different exit code (3 if memory serves), since you ideally don't want to quit here.